### ISSUE TITLE
Profiles: Workbench startup scaffolding should 

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#153457]
Every time I open a workspace on a different profile, in which I have a different theme, the workbench starts up with the wrong colors:

![Recording 2022-06-28 at 14 15 46 (1)](https://user-images.githubusercontent.com/22350/176176133-f88761fe-0165-4165-8196-d752ee3a3ac9.gif)

Should we save these base colors per profile?

**Comment by jrieken** [comment]
Agree - this is stored in a special way and needs some special treatment

**Comment by jrieken** [comment]
Yeah, let spend some time during debt week. The state is actually store in a dedicated file, not in normal storage etc
